man/tpm2_dictionarylockout.8.in

```diff
@@ -0,0 +1,69 @@
+.\" Copyright (c) 2016, Intel Corporation
+.\" All rights reserved.
+.\"
+.\" Redistribution and use in source and binary forms, with or without
+.\" modification, are permitted provided that the following conditions are met:
+.\"
+.\" 1. Redistributions of source code must retain the above copyright notice,
+.\" this list of conditions and the following disclaimer.
+.\"
+.\" 2. Redistributions in binary form must reproduce the above copyright notice,
+.\" this list of conditions and the following disclaimer in the documentation
+.\" and/or other materials provided with the distribution.
+.\"
+.\" 3. Neither the name of Intel Corporation nor the names of its contributors
+.\" may be used to endorse or promote products derived from this software without
+.\" specific prior written permission.
+.\"
+.\" THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+.\" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+.\" IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+.\" ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+.\" LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+.\" CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+.\" SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+.\" INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+.\" CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+.\" ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF
+.\" THE POSSIBILITY OF SUCH DAMAGE.
+.TH tpm2_dictionarylockout 8 "February 2017" Intel "tpm2.0-tools"
+.SH NAME
+tpm2_dictionarylockout\ - setup dictionary-attack-lockout parameters or clear dictionary-attack-lockout state, if any passwd option is missing, assume NULL.
+.SH SYNOPSIS
+.B tpm2_dictionarylockout[ COMMON OPTIONS ] [ TCTI OPTIONS ] [ \fB\-\-setup-parameters\fR|\fB\-\-clear-lockout\fR|\fB\-\-lockout-passwd\fR|\fB\-\-lockout-recovery\fR|\fB\-\-recovery-time\fR|\fB\-\-max-tries\fR|\fB ]
+.PP
+setup dictionary-attack-lockout parameters or clear dictionary-attack-lockout state, if any passwd option is missing, assume NULL.
+.SH DESCRIPTION
+.B tpm2_dictionarylockout
+setup dictionary-attack-lockout parameters or clear dictionary-attack-lockout state, if any passwd option is missing, assume NULL. 
+.SH OPTIONS
+.TP
+\fB\-S ,\-\-setup-parameters\fR
+specifies the tool should operate to setup dictionary-attack-lockout parameters.
+.TP
+\fB\-a ,\-\-clear-lockout\fR
```

Comment:
A see a capital -C option below, but I don't see it mentioned here or part of the common options. Perhaps -c for clear would make more sense than -a?